shared/types/background.ts

```diff
@@ -125,9 +125,6 @@ export type ControllerStatePropertiesEnumerated = {
   lastInteractedConfirmationInfo?: AppStateControllerState['lastInteractedConfirmationInfo'];
   termsOfUseLastAgreed?: AppStateControllerState['termsOfUseLastAgreed'];
   snapsInstallPrivacyWarningShown?: AppStateControllerState['snapsInstallPrivacyWarningShown'];
-  interactiveReplacementToken?: AppStateControllerState['interactiveReplacementToken'];
```

Comment:
These properties were only set and only used on MMI builds, despite not being fenced here.